components/space-pathway-profile.jsx

```diff
@@ -0,0 +1,34 @@
+var React = require('react');
+
+var SpacePathwayProfile = React.createClass({
+  render: function() {
+    return (
+      <div className="space-pathway-profile">
+        <div className="detail">
+          <div className="header">
+            <h1>{this.props.name}</h1>
+          </div>
+          { this.props.type ? <div className="type">{this.props.type}</div> : null }
+          <div className="description" dangerouslySetInnerHTML={{__html: this.props.description}}></div>
```

Comment:
does `description` contain HTML code? If not, let's not use this. If it does: that feels dangerous, why does it contain raw HTML?